boards/arm/ks_zion/doc/index.rst

```diff
@@ -0,0 +1,127 @@
+.. _ks_zion_board:
+
+
+Kernelspace Zion
+################
+
+.. contents::
+
+
+Overview
+========
```

Comment:
Please apply https://docs.zephyrproject.org/latest/contribute/documentation/guidelines.html#headings throughout the file (should be `*****`s here, for example) -- again, looking at other boards would provide you with the right framework